cypher/cypher-prettifier/src/test/scala/org/neo4j/cypher/docgen/tooling/PrettifierParserTest.scala

```diff
@@ -82,23 +105,31 @@ class PrettifierParserTest extends ParserTestBase[Seq[SyntaxToken], Seq[SyntaxTo
     }
   }
 
-  test("shouldParseDropConstraintByName") {
```

Comment:
Moved this test up above `shouldParseShowConstraints` to be consistent with the index tests, it has confused github